packages/core/src/command.ts

```diff
@@ -316,10 +318,16 @@ export class Command<T extends AvailableCommandOption> {
     let chain: Promise<any> = Promise.resolve();
 
     if (this.commandName !== 'info') {
+      this.options.isIndependent = this.project.isIndependent();
       chain = chain.then(() => this.project.getPackages());
       chain = chain.then((packages) => {
         const { graphType } = this.options.command?.[this.commandName] ?? {};
         this.packageGraph = new PackageGraph(packages || [], graphType ?? 'allDependencies');
+        if (this.options.isIndependent) {
```

Comment:
shouldn't that also be working with fixed mode and not just independent?

Reply:
@ghiscoding yes i can make it work in both cases. But does it really makes sense? Because if you use version with scope in fixed mode then you end up with a package with a different version than others.
This is the only reason why i added that if case

Reply:
I think it does but in very rare occasion like maybe patch versions or other things that I can't think of